x-pack/plugins/security_solution/public/detection_engine/rule_exceptions/components/edit_exception_flyout/index.tsx

```diff
@@ -335,6 +348,14 @@ const EditExceptionFlyoutComponent: React.FC<EditExceptionFlyoutProps> = ({
     expireTime,
   ]);
 
+  const handleOnSubmit = useCallback(() => {
```

Comment:
Same as above 👍 
But also - this seems to be a duplicated code, can we do something about the reusability here?

Reply:
I'm not sure that makes sense here since it's using variables from the individual reducers.